iam.tf

```diff
@@ -34,10 +34,12 @@ data "aws_iam_policy_document" "task_execution_cloudwatch_access" {
   statement {
     effect = "Allow"
     actions = [
-      "logs:PutRetentionPolicy",
-      "logs:CreateLogGroup"
+      # "logs:PutRetentionPolicy",
+      # "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
     ]
-    resources = ["arn:aws:logs:${var.aws_region}:${local.aws_account_id}:log-group:${module.label.id}:*"]
+    resources = ["arn:aws:logs:${var.aws_region}:${local.aws_account_id}:log-group:${local.cloudwatch_log_group_name}:*"]
```

Comment:
Is it possible to have multiple resources arn values crafted by cw log group use case ?
* cloudwatch_log_group_create = false -> [keep log-group arn generated by ECS itself](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html)
*  cloudwatch_log_group_create=false & cloudwatch_log_group_name="" -> use module.lable.id generated log group
* cloudwatch_log_group_create=false & cloudwatch_log_group_name!="" -> use user defined log group name

Reply:
did you mean `cloudwatch_log_group_create=true` with the 2nd and the 3rd use case?

because I can't see the difference between the 1st use case and the others if all use cases have `cloudwatch_log_group_create = false`